tcl/org.testeditor.tcl.dsl.tests/src/org/testeditor/tcl/dsl/validation/TclParameterUsageValidatorTest.xtend

```diff
@@ -140,26 +101,26 @@ class TclParameterUsageValidatorTest extends AbstractParserTest {
 				macros += macro("MyCallMacro") => [
 					template = template("mycall").withParameter("unknown")
 					contexts += macroTestStepContext(macroModel.macroCollection) => [
-						step = testStep("othercall").withVariableReference("unknown")
+						step = testStep("othercall").withReferenceToTemplateVariable("unknown")
 					]
 				]
 				macros += macro("OtherCallMacro") => [
 					template = template("othercall").withParameter("secs")
 					contexts += componentTestStepContext(dummyComponent) => [
-						steps += testStep("wait").withVariableReference("secs") // secs are expected to be of type long in aml fixture
+						steps += testStep("wait").withReferenceToTemplateVariable("secs") // secs are expected to be of type long in aml fixture
 					]
 				]
 			]
 		]
-		macroModel.register("tml")
+		macroModel.register("test.tml")
 
 		// call "mycall" with env parameter (which is of type String, transitively expected is type long) ...
 		val tclModel = tclCallingMyCallMacroWithOneEnvParam("myEnvString", macroModel)
 
 		val myCallContext = tclModel.test.steps.head.contexts.head
 
 		// when
-		val setWithLong = tclValidator.getAllTypeUsagesOfVariable(myCallContext, "myEnvString")
+		val setWithLong = tclModelUtil.getAllTypeUsagesOfVariable(myCallContext, "myEnvString")
```

Comment:
Seems like this would test `TclModelUtil`. If so this test should be moved to `TclModelUtilTest`.